3회차/문재윤/0_최빈수구하기.py

```diff
@@ -1,3 +1,24 @@
 import sys
 
 sys.stdin = open("_최빈수구하기.txt")
+n = int(input())
+
+for j in range(n):
+    f = int(input())
+    n = list(input().split())
+    nn = set(n)
+    a = 0
+    answer1 = 0
+    for i in nn:
+        count_ = n.count(i)
+        if int(count_) > a:
```

Comment:
`count` 함수의 반환 값은 이미 int에요! 별도로 `int` 함수를 씌어줄 필요가 없겠죠?
```suggestion
        if count_ > a:
```